packages/components/src/local/organisms/setting-platform-types/index.tsx

```diff
@@ -88,8 +89,22 @@ export const SettingPlatformTypes: React.FC<PropTypes> = ({ platformType, onChan
   }
 
   const handleDelete = (item: Item): void => {
+    const db: PouchDB.Database = new PouchDB(databasePath + wargameName + dbSuffix)
```

Comment:
Hmm, @lilitkarapetyan - this feels like a bit of a bad code-smell.  It's only when `SAVE` is clicked that data goes to the database.  So, updating the client state should be enough.

Can you help me understand why we have the database round-trip?